src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -353,9 +353,12 @@ const ChartHeader = styled('header')`
   margin-bottom: ${space(1)};
   display: grid;
   grid-template-columns: 1fr max-content;
-`;
+  grid-gap: ${space(2)};
 
-const ChartActions = styled('div')``;
+  > div:first-of-type {
```

Comment:
I think using a styled component would be more direct and clear than a child and positional selector.